### DISCUSSION TITLE
How to download official supported WRF-Hydro V5.2.0 ？

### DISCUSSION THREAD

**Discussion by andrewsoong** [NCAR/wrf_hydro_nwm_public#567] [Q&A] [answered]
How to download official supported WRF-Hydro V5.2.0 as is described by WRF4.3 released in 05-10,2021 ？
https://github.com/NCAR/wrf_hydro_nwm_public site supplies pre-release V5.2.0（un-official supported）.

**Comment by mollymca** [comment, marked as answer]
Hi Andrew, 
5.2.0 is not officially publicly supported at this time. Follow @WRFHydro on twitter or our user forum or subscribe to our email list to be informed. https://ral.ucar.edu/projects/wrf_hydro/subscribe

**Reply by HathewayWill** [reply]
Page is dead

**Comment by andrewsoong** [comment]
> Hi Andrew,
> 5.2.0 is not officially publicly supported at this time. Follow @WRFHydro on twitter or our user forum or subscribe to our email list to be informed. https://ral.ucar.edu/projects/wrf_hydro/subscribe

OK! Thank you! When will you release officially supported v5.2.0?
